reflex/components/component.py

```diff
@@ -677,7 +677,7 @@ def _add_style(self, style: dict):
         """
         self.style.update(style)
 
-    def add_style(self, style: ComponentStyle) -> Component:
+    def add_style_recursive(self, style: ComponentStyle) -> Component:
```

Comment:
Let's call it `_add_all_styles` for consistency

Reply:
look at what the function does, it's not adding all styles, it's adding the style in the args to itself and children.

Reply:
also, are we adding the prefix here? In the ticket, it was not adding that.

Reply:
Got it, add_style_recursive is good then. I think we should add the prefix because this isn't a method we expect the user to have to modify

Reply:
In a way it is adding all styles in the style dict recursively down the tree, so I think add_all_styles makes sense as well.